### ISSUE TITLE
Metrics in the API

### ISSUE THREAD

**Issue by Shibumi497** [nilmtk/nilmtk#863]
Hi everybody,

For my comparison,  the performance metrics are essentials and I would like to use the most I can (Of  both Energy estimation and event detection)
I would like to implement more metrics in the API, or to use the metrics from sklearn at least,  do you know how can I make it out?

Thank you in advance 

I identify the ones I see often answering
@PMeira @nipunbatra

**Comment by Shibumi497** [comment]
There are news!
I entered into API and Losses.py , I imported F1 score and relative error, it works well.
However, when I try to import others from sklearm.metrics,  and even for the precision score (that is a part of the calculation of the F1 score...), it doesn't work and return me "Loss function  precision  is not supported currently!" for example.

How can I make it works?
